client/blocks/importer/wordpress/upgrade-plan/hooks/use-get-upgrade-plan-site-metrics.tsx

```diff
@@ -0,0 +1,39 @@
+import { useIsEnglishLocale } from '@automattic/i18n-utils';
+import { getQueryArg } from '@wordpress/url';
+import { useUrlBasicMetricsQuery } from 'calypso/data/site-profiler/use-url-basic-metrics-query';
+import { upgradePlanSiteMetricsLcpThreshold } from '../constants';
+
+export const useGetUpgradePlanSiteMetrics = () => {
+	const isEnglishLocale = useIsEnglishLocale();
+	const importSiteQueryParam = getQueryArg( window.location.href, 'from' )?.toString() || '';
+	const { data: siteMetricData } = useUrlBasicMetricsQuery( importSiteQueryParam );
+	let showUpdatedSpeedMetrics = false;
+	let lcpPercentageDifference = 0;
+
+	if (
+		isEnglishLocale &&
+		siteMetricData?.basic?.lcp &&
+		siteMetricData?.basic?.lcp > upgradePlanSiteMetricsLcpThreshold
+	) {
+		lcpPercentageDifference =
```

Comment:
While working on https://github.com/Automattic/wp-calypso/pull/90717, I noticed something with the calc.

Maybe I'm missing something, but I think if the LCP of a site is 5000ms, we should consider that the sites with 2500ms (the threshold) are 50% faster, right? Or am I missing something (maybe I misunderstood something related to the other questions I sent in this PR)?

If I'm correct, I think we could use something like this for the calc:

```js
const lcpMsDiff = siteMetricData?.basic?.lcp - upgradePlanSiteMetricsLcpThreshold;
lcpPercentageDifference = Math.round( ( lcpMsDiff / siteMetricData?.basic?.lcp ) * 100 );
```

Reply:
Wouldn't 2500ms -> 5000ms be a 100% increase?
Maybe the formula can be something like this ( divide by the threshold):

```
const lcpMsDiff = siteMetricData?.basic?.lcp - upgradePlanSiteMetricsLcpThreshold;
lcpPercentageDifference = Math.round( ( lcpMsDiff / upgradePlanSiteMetricsLcpThreshold ) * 100 );
```

Reply:
> Wouldn't 2500ms -> 5000ms be a 100% increase?

If we say that their site is 100% slower, I think it would be accurate. But since we say the opposite (how faster the other WPCOM sites are), I think it would be 50% faster.

100% faster, I think it would be a site that loads immediately (0ms).

Does it make sense?

Which makes me think that since the bigger number would be saying the opposite, maybe we should change the text for a bigger impact? _"Your site is at least X% slower than 83% of sites on WordPress.com."_

Reply:
Yeah, that makes sense. 

>"Your site is at least X% slower than 83% of sites on WordPress.com."
A similar text was suggested (p55Cj4-2Wq-p2#comment-3467), but it seems like we're aiming to phrase it from the perspective of WordPress so this may not be ideal.

Reply:
Thanks for the context, @agrullon95! So if we want to keep that phrase, we just need to update the calc?